ProjectHive/ProjectHive.Services.NotificationsAPI/Services/WebSocketHandler.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Newtonsoft.Json;
+using ProjectHive.Services.NotificationsAPI.Dto;
+using ProjectHive.Services.NotificationsAPI.Models;
+using System.Net.WebSockets;
+using System.Text;
+
+namespace ProjectHive.Services.NotificationsAPI.Services
+{
+    public class WebSocketHandler :IWebSocketHandler
+    {
+        private readonly List<WebSocket> _sockets = new List<WebSocket>();
+
+        public async Task SendNotificationAsync(NotificationDto notification)
+        {
+            var message = JsonConvert.SerializeObject(notification);
+            var buffer = Encoding.UTF8.GetBytes(message);
+            var segment = new ArraySegment<byte>(buffer);
+
+            foreach (var socket in _sockets)
+            {
+                if (socket.State == WebSocketState.Open)
+                {
```

Comment:
There's no error handling in your SendNotificationAsync method. Please start to use some to prevent the app crashes

Wrap your WebSocket send operation in a try-catch block and log any exceptions that occur.